src/components/slider/slider.spec.js

```diff
@@ -136,6 +136,162 @@ describe('md-slider', function() {
+      expect(slider.attr('aria-valuemax')).toEqual('7');
+    });
+
+    it('should have updated value correctly', function () {
+      expect(slider.attr('aria-valuenow')).toEqual('6');
+    });
+
+  });
+
+  describe('when updating max and model value below previous max simultaneously', function() {
+
+    var slider = null;
+
+    beforeEach(function() {
+      slider = setup('min="0" max="{{max}}" ng-model="model"')
+      pageScope.max = 4;
+      pageScope.model = 2;
+      pageScope.$apply();
+      expect(slider.attr('aria-valuenow')).toEqual('2');
+      expect(slider.attr('aria-valuemax')).toEqual('4');
+      pageScope.model = 3;
+      pageScope.max = 5;
+      pageScope.$apply();
+    });
+
+    it('should have updated max correctly', function () {
+      expect(slider.attr('aria-valuemax')).toEqual('5');
+    });
+
+    it('should have updated value correctly', function () {
+      expect(slider.attr('aria-valuenow')).toEqual('3');
+    });
+
+  });
+
+  describe('when updating min and model value equally below previous min simultaneously', function() {
+
+    var slider = null;
+
+    beforeEach(function() {
+      slider = setup('min="{{min}}" max="10" ng-model="model"')
+      pageScope.min = 5;
+      pageScope.model = 5;
+      pageScope.$apply();
+      expect(slider.attr('aria-valuenow')).toEqual('5');
+      expect(slider.attr('aria-valuemin')).toEqual('5');
+      pageScope.model = 2;
+      pageScope.min = 2;
+      pageScope.$apply();
+    });
+
+    it('should have updated min correctly', function () {
+      expect(slider.attr('aria-valuemin')).toEqual('2');
+    });
+
+    it('should have updated value correctly', function () {
+      expect(slider.attr('aria-valuenow')).toEqual('2');
+    });
+
+  });
+
+  describe('when updating min and model value below previous min simultaneously', function() {
+
+    var slider = null;
+
+    beforeEach(function() {
+      slider = setup('min="{{min}}" max="10" ng-model="model"')
+      pageScope.min = 5;
+      pageScope.model = 6;
+      pageScope.$apply();
+      expect(slider.attr('aria-valuenow')).toEqual('6');
+      expect(slider.attr('aria-valuemin')).toEqual('5');
+      pageScope.model = 3;
+      pageScope.min = 2;
+      pageScope.$apply();
+    });
+
+    it('should have updated min correctly', function () {
+      expect(slider.attr('aria-valuemin')).toEqual('2');
+    });
+
+    it('should have updated value correctly', function () {
+      expect(slider.attr('aria-valuenow')).toEqual('3');
+    });
+
+  });
+
+  describe('when updating min and model value above previous min simultaneously', function() {
+
+    var slider = null;
+
+    beforeEach(function() {
+      slider = setup('min="{{min}}" max="10" ng-model="model"')
+      pageScope.min = 5;
+      pageScope.model = 6;
+      pageScope.$apply();
+      expect(slider.attr('aria-valuenow')).toEqual('6');
+      expect(slider.attr('aria-valuemin')).toEqual('5');
+      pageScope.model = 5;
```

Comment:
This seems to be checking "when updating min and model value equal to previous min simultaneously" rather than "when updating min and model value above previous min simultaneously". Since the previous min was `5` and the new model value is `5`. Am I reading this wrong? Was this intended?

Reply:
I agree, this was not entirely clear so I updated both the title and expectations of the test to better match the intention.